cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/keyspaces/CradleInfoKeyspaceCreator.java

```diff
@@ -24,21 +24,39 @@
 import com.datastax.oss.driver.api.querybuilder.SchemaBuilder;
 import com.exactpro.cradle.cassandra.CassandraStorageSettings;
 import com.exactpro.cradle.cassandra.utils.QueryExecutor;
+import com.exactpro.cradle.utils.CradleStorageException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class CradleInfoKeyspaceCreator extends KeyspaceCreator
 {
+	private static final Logger logger = LoggerFactory.getLogger(CradleInfoKeyspaceCreator.class);
+	
 	public CradleInfoKeyspaceCreator(QueryExecutor exec, CassandraStorageSettings settings)
 	{
 		super(settings.getCradleInfoKeyspace(), exec, settings);
 	}
-	
+
 	@Override
 	protected void createTables() throws IOException
 	{
 		createBooks();
 	}
-	
-	
+
+	@Override
+	public void createAll() throws IOException, CradleStorageException
+	{
+		try
+		{
+			createKeyspace();
+			createTables();
+		}
+		catch (CradleStorageException e)
+		{
+			logger.info("Existing \"Cradle Info\" keyspace '{}' is in use", getKeyspace());
```

Comment:
Hiding exceptions can lead to troubles. Maybe, createKeyspace should check for presence of keyspace and not create it if not necessary? Or exception should be thrown to be handled outside of API.